en/asgardeo/mkdocs.yml

```diff
@@ -313,44 +313,64 @@ nav:
-      - Export user information API: apis/register-mfa/export-user-info.md
-    - Administrators:
-      - Export admin information API: apis/administrators/export-admin-info.md
-    - Consent management API: apis/consent-management.md
-    - Attribute management API: apis/attribute-management.md
-    - Validation API: apis/validation.md
-    - Idle account identification API: apis/idle-account-identification.md
-    - B2B self service management API: apis/onboard-admins-self-service.md
-    - Organization management API: apis/organization-management.md
-    - Organization discovery configuration management API: apis/organization-discovery-config-mgt-rest-api.md
-    - Organization APIs:
-      - Organization APIs: apis/organization-apis/index.md
+    - System APIs:
+      - Admin advisory management API:  apis/admin-advisory-management-rest-api.md
+    - Management APIs:
+      - API resource management API: apis/api-resource-management-rest-api.md
+      - Application management API: apis/application-management.md
+      - Attribute management API: apis/attribute-management.md
+      - Authenticators API: apis/authenticators.md
+      - Branding Preferences API: apis/branding-preference-rest-api.md
+      - Event configuration API: apis/event-configuration.md
+      - Identity provider API: apis/idp.md
+      - Idle account identification API: apis/idle-account-identification.md
+      - OAuth Dynamic Client Registration API: apis/oauth-dcr.md
+      - Organization discovery API: apis/organization-discovery.md
+      - Organization discovery configuration management API: apis/organization-discovery-config-mgt-rest-api.md
+      - Organization management API: apis/organization-management.md
+      - Password expiring users identification API: apis/password-expiring-users-identification.md
+      - Role management (SCIM 2.0) API: apis/roles.md
+      - Server management:
+        - Identity governance API: apis/identity-governance.md
+        - Consent management API: apis/consent-management.md
+        - Email templates API: apis/email-template.md
+      - User Functionality management API: apis/functionality.md
+      - User management:
+        - SCIM 2.0 API: apis/scim2.md
+        - User account associations API: apis/association-management-by-admin.md
+        - Account Recovery API: apis/user-account-recovery.md
+        - Offline user onboard management API: apis/offline-user-onboard.md
+      - User sessions:
+        - Session management API: apis/session.md
+        - Session extension API: apis/extend-sessions.md
+      - Validation API: apis/validation.md
+    - Orgnization APIs:
+      - Orgnization APIs: apis/organization-apis/index.md
+      - Application management API: apis/organization-apis/org-application-management.md
+      - Authenticators API: apis/organization-apis/org-authenticators.md
+      - B2B self-service management API: apis/onboard-admins-self-service.md
```

Comment:
This API is not promoting, let's remove

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/906ac77a43915626b50b0feb4782769a4418d87e